test/layers/basic_tests.jl

```diff
@@ -87,7 +87,7 @@
                     atol = 5 * eps(Float32)
                 ))
 
-            @jet layer(x, ps, st)
+            @jet layer(x, ps, st) target_modules = (Lux, LuxCore, LuxLib)
```

Comment:
No you don't need to specify this here, `@jet` uses the Preferences file to pick this up. But for testing locally you will have to specify them

Reply:
Gotcha. I'll revert that commit.